.travis.yml

```diff
@@ -2,6 +2,7 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y libzmq3-dev
   - sudo -E ./install-protobuf.sh
+  - gem update --system
```

Comment:
This is the solution to a rainbow installation issue... dunno when we'll be able to get rid of it

https://github.com/sickill/rainbow/issues/44